Fixes TARGET_THEM to use target_gender.him

Runa-Dacino opened the pull request:
Partly fixes https://github.com/Baystation12/Baystation12/issues/15035

* Fixes TARGET_THEM to use target_gender.him

TARGET_THEM used target_gender.he which caused emotes to appear as

"gives dabs to heself" and "gives dabs to sheself". Fix makes it appear as "gives dabs to himself" and "gives dabs to herself."

* Creates changelog. This time with .yml ending from getgo

<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You find a README and example file in .\html\changelogs\ for further instructions.
-->

Runa-Dacino committed:
Fixes TARGET_THEM to use target_gender.him

* Fixes TARGET_THEM to use target_gender.him

TARGET_THEM used target_gender.he which caused emotes to appear as

"gives dabs to heself" and "gives dabs to sheself". Fix makes it appear as "gives dabs to himself" and "gives dabs to herself."

* Creates changelog. This time with .yml ending from getgo

Techhead0 commented on html/changelogs/RunaDacino-EmoteGenderFix.yml:
```diff
@@ -0,0 +1,36 @@
+#################################
+
+# Your name.  
+author: N3X15
```
Don't want the credit, Runa?

Runa-Dacino replied:
>.> Fixing!

Runa-Dacino committed:
Credited myself instead of N3X15

Why can't I ever get a nice and squashed single commit PR? I even had .yml ending this time :(
